draft-ietf-webpush-protocol.xml

```diff
@@ -530,10 +558,11 @@ TTL = 1*DIGIT
       </t>
       <t>
         Each push message is pushed in response to a synthesized GET request.
-        The GET request is made to the push message resource that was created by
+        This GET request is made to the push message resource that was created by
         the push service when the application server requested message delivery.
-        The response body is the entity body from the most recent request sent
-        to the push resource.
+        The push service MUST include the link references to the push and receipt
+        subscribe resources in the request. The response body is the entity body

```

Comment:
To be clear, it is not required. The original text placed it in the response as a SHOULD:

*In addition, the push service SHOULD return link references to the push and receipt subscribe resources.*

From an implementation perspective, I thought it would be easier to return these link relations together. 

Frankly, I would still like to NOT return the receipt subscribe resource, but we can make the separation between returning the :push link relation in the PUSH_PROMISE as a MUST and returning the :push:receipt link relation in the pushed response as a SHOULD.